bin/inference/pycbc_inference_plot_movie

```diff
@@ -73,19 +74,22 @@ logging.info('Loading parameters')
 fp, parameters, labels, _ = option_utils.results_from_cli(opts,
                             load_samples=False)
 
+if opts.end_sample is None:
+    opts.end_sample = fp.niterations
 # Define thin interval based on number of frames or frame step
 if opts.frame_number and opts.frame_step:
     raise ValueError("Only one of frame-number or frame-step must be provided.")
 elif opts.frame_number:
     nframes = opts.frame_number
-    thinint = (fp.niterations / (nframes - 1)) - 1
+    thinint = (opts.end_sample - opts.start_sample)/nframes
```

Comment:
Have you also tried without specifying start_sample and end_sample? A command line in the same way it would be run with the old version?